src/app/core/services/mileage-rates.service.ts

```
@@ -97,20 +99,23 @@ export class MileageRatesService {
+      four_wheeler3: this.translocoService.translate('services.mileageRates.fourWheelerType3'),
+      four_wheeler4: this.translocoService.translate('services.mileageRates.fourWheelerType4'),
+      bicycle: this.translocoService.translate('services.mileageRates.bicycle'),
+      electric_car: this.translocoService.translate('services.mileageRates.electricCar'),
     };
 
     return rateName && names[rateName] ? names[rateName] : rateName;
   }
 
   getReadableRate(rate: number, currency: string, unit: string): string {
-    unit = unit && unit.toLowerCase() === 'miles' ? 'mile' : 'km';
+    unit =
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)